src/main/java/com/minecolonies/coremod/colony/buildings/BuildingHome.java

```diff
@@ -251,7 +216,7 @@ private void addHomelessCitizens()
             // Move the citizen to a better hut
             if (citizen.getHomeBuilding() != null && citizen.getHomeBuilding().getBuildingLevel() < this.getBuildingLevel())
             {
-                if(citizen.getHomeBuilding() instanceof BuildingHome)
+                if (citizen.getHomeBuilding() instanceof BuildingHome)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1066)